deepchem/models/torch_models/layers.py

```diff
@@ -14,36 +15,175 @@ class ScaleNorm(nn.Module):
   Finally, the result is returned as `input_tensor * norm value`.
 
   This layer can be used instead of LayerNorm when a scaled version of the norm is required.
-  Instead of performing the scaling operation (scale / norm) in a lambda-like layer, we are defining it within this layer to make prototyping more efficient.
+  Instead of performing the scaling operation (`scale / norm`) in a lambda-like layer, we are defining it within this layer to make prototyping more efficient.
 
   References
   ----------
   .. [1] Lukasz Maziarka et al. "Molecule Attention Transformer" Graph Representation Learning workshop and Machine Learning and the Physical Sciences workshop at NeurIPS 2019. 2020. https://arxiv.org/abs/2002.08264
-  
+
   Examples
   --------
   >>> import deepchem as dc
   >>> scale = 0.35
-  >>> layer = dc.models.torch_models.ScaleNorm(scale)
+  >>> layer = dc.models.torch_models.layers.ScaleNorm(scale)
   >>> input_tensor = torch.tensor([[1.269, 39.36], [0.00918, -9.12]])
-  >>> output_tensor = layer.forward(input_tensor)
+  >>> output_tensor = layer(input_tensor)
   """
 
-  def __init__(self, scale, eps=1e-5):
+  def __init__(self, scale: int, eps: float = 1e-5):
     """Initialize a ScaleNorm layer.
 
     Parameters
     ----------
     scale: Real number or single element tensor
       Scale magnitude.
     eps: float
-      Epsilon value.
+      Epsilon value. Default = 1e-5.
```

Comment:
We usually do something like:

```
eps: float (default 1e-5)
  Epsilon value
```

Reply:
Changed! I've added this change to this branch (`GeneratorPR`)